test/ValoremOptionsClearinghouse.unit.t.sol

```diff
@@ -992,20 +992,6 @@ contract ValoremOptionsClearinghouseUnitTest is BaseClearinghouseTest {
         vm.stopPrank();
     }
 
-    function testRevert_redeem_whenClaimTooSoon() public {
```

Comment:
This test should not go away, but rather be altered to catch the remaining cases.

Reply:
It moved to the other 1.1 test file.

Reply:
My thinking is it will be cleaner for the auditors to encapsulate all 1.1 functionality in its own test class. Open to otherwise tho.